app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -1,6 +1,6 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
-import { KeyboardShortcuts } from '../../shared/components/useKeyboardShortcuts';
```

Comment:
Any reason for removing keyboard shortcuts?

Reply:
The Escape keypress can be handled by the FocusTrap lib so I'm just deferring to that instead of pulling in the extra KeyboardShortcodes 👍

Reply:
Oh, awesome! :)